### PR TITLE
feat: Mock EDV APIs: Create Vault, Store/Retrieve Document - Simple in-memory DB

### PR DESCRIPTION
**Pull request by DRK3**
Implementation of Create Vault and Store/Retrieve document API calls. Everything is unencrypted and stored in an in-memory database for now.

Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: cmd/edv-rest/startcmd/start.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+const (
+	hostURLFlagName      = "host-url"
+	hostURLFlagShorthand = "u"
+	hostURLFlagUsage     = "URL to run the edv instance on. Format: HostName:Port."
+	hostURLEnvKey        = "EDV_HOST_URL"
+)
+
+var errMissingHostURL = errors.New("host URL not provided")
+
+type edgeStoreParameters struct {
```
Comment (fqutishat) [inline cmd/edv-rest/startcmd/start.go]: rename it

File: pkg/utils/cmd/util_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	hostURLFlagName      = "host-url"
+	hostURLFlagShorthand = "u"
+	hostURLFlagUsage     = "URL to run the edge-auth instance on. Format: HostName:Port."
```
Comment (fqutishat) [inline pkg/utils/cmd/util_test.go]: rename it

File: cmd/edv-rest/startcmd/start.go (hunk 1 above)
Comment (DRK3) [reply cmd/edv-rest/startcmd/start.go]: Done

File: pkg/utils/cmd/util_test.go (hunk 2 above)
Comment (DRK3) [reply pkg/utils/cmd/util_test.go]: Done

**Pull request merged**